feat: add entities for referentiel connector

AlexMyddleware opened the pull request:
1st pull request !

AlexMyddleware committed:
feat:added tables

added tables InternalList and InternalListFeature
for future connector
Signed-off-by: AlexMyddleware <alexistroitzky@myddleware.com>

stlgaits commented on src/Entity/InternalList.php:
```diff
@@ -0,0 +1,157 @@
+use App\Repository\InternalListRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+//test git
```
Avoid leaving dev comments ^^

AlexMyddleware committed:
feat: remove unnecessary comment

stlgaits reviewed (approved):
Looks good to me

AlexMyddleware committed:
fix: update table InternalListValue

update table InternalListValue to include relation OneToMany instead of OneToOne for listid attribute

Signed-off-by: AlexMyddleware <alexistroitzky@myddleware.com>

AlexMyddleware committed:
fix: update table InternalListValue

update tables configuration: InternalList relation corrected to ManyToOne

AlexMyddleware committed:
feat: create Solution internallist.php

WIP, Solution not functional

AlexMyddleware committed:
feat: internallist connector

edit Internallist and InternallistValue entities so as to handle relation management
add custom placeholder icon for internallist connector
WIP connector detected

AlexMyddleware committed:
feat: add login form for internallist.php

and add metadata for internallist connector

stlgaits commented:
I am closing this PR as the code is already present in Release-3.1.8 PR https://github.com/Myddleware/myddleware/pull/810
